pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -54,6 +54,9 @@ const (
 
 // AzVolumeSpec is the spec for an AzVolume resource
 type AzVolumeSpec struct {
+	// the name of PV that corresponds to the AzVolume obj
+	//+optional
+	PersistentVolume string `json:"persistentVolume"`
```

Comment:
Since this isn't set by the `CrdProvisioner` but rather by the `pv` controller as part of its reconciliation cycle, I believe this should be part of the `Status` field rather than the `Spec` field.